### PR TITLE
Adding sentry package

### PR DESCRIPTION
**Pull request by CXZ7720**
Adding sentry to follow client error more easily

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -4,29 +4,38 @@ import Vue from 'vue'
...
+});
 
 var VueCookie = require('vue-cookie');
 
 Vue.use(VueGtag, {
-  config: {
-    id: 'UA-167215120-1'
-  }
+    config: {
+        id: 'UA-167215120-1'
```
Comment (Taewan-P) [inline src/main.js]: Is there any reason to add additional indentation here? Also applies to Line 18,20,37,38,39,40.

**Review by Taewan-P — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (CXZ7720) [reply src/main.js]: I didn't mean to indent it, but auto reformatting function makes some additional indent here. I'll recheck about it with next commit.

File: src/main.js [hunk 2]
```diff
@@ -19,14 +26,16 @@ Vue.config.productionTip = false
...
+          console.log("Service Worker Registered.");
+        });
   });
 }
 /* eslint-disable no-new */
 new Vue({
   el: '#app',
   router,
-  components: { App },
+  components: {App},
```
Comment (Taewan-P) [inline src/main.js]: Indentation bool편

Comment (CXZ7720) [reply src/main.js]: 엌

**Review by Taewan-P — COMMENTED** [review]
Indentation problem in main.js, I wouldn't be too picky. You can either change it or leave it.

**Review by Taewan-P — APPROVED** [review]
Thank you. Great work

**Pull request merged**